tests/test-stack-trace.js

```diff
@@ -124,6 +136,14 @@ var testDefs = [
     limit: 1
   },
 
+  { name: 'top most webppl entry is not a user function',
+    code: ['var f = function() { stackTraceTestFns.test() };',
```

Comment:
No, you're right, I messed up! That test function used to be in headerUtils where `s,k,a` made sense, but when I switched to temporarily attaching it to `global` I broke things. Actually, I'm now not entirely sure it was right before that, although as written the test did fail before the fix, and pass after. Either way, I'll push a fix shortly. Thanks!